ChangeLog.d/updated_windows_apis.txt

```diff
@@ -0,0 +1,6 @@
+Requirement changes
+   * Update Windows APIs to use BCryptGenRandom and wcslen and
+     ensure that conversions between size_t, ULONG, and int are
+     always done safely. Minimum required Windows version is
+     Windows Vista. Original contribution by Kevin Kane #635,
```

Comment:
I'm going to be picky, and say "can we put the change to minimum required Windows version first, please, ideally with the word 'now'?".

It's probably best to separate out into the changes, and then the requirement changes.

And we didn't really update "Windows APIs" - we updated the Windows-specific code to use more modern Windows APIs, so maybe something like


```
Requirement changes
   * Minimum required Windows version is now Windows Vista.

Changes
   * Update Windows code to use BCryptGenRandom and wcslen, and
     ensure that conversions between size_t, ULONG, and int are
     always done safely.  Original contribution by Kevin Kane #635, #730
     followed by Simon Butcher #1453.
```